packages/o/opencl/xmake.lua

```diff
@@ -96,6 +96,9 @@ package("opencl")
 
         local configs = {"-DOPENCL_SDK_BUILD_SAMPLES=OFF"}
         table.insert(configs, "-DCMAKE_BUILD_TYPE=" .. (package:debug() and "Debug" or "Release"))
+        if package:is_plat("macosx") and package:is_arch("x86_64") then 
+            package:addenv("DYLD_LIBRARY_PATH", path.join(package:installdir(), "lib"))
```

Comment:
please do not add it.

if you use shared library, DYLD_LIBRARY_PATH will be added automatically.

```lua
add_requires("opencl", {configs = {shared = true}})
```

Reply:
https://github.com/xmake-io/xmake/blob/22455f25e6e61438ab7e3d5766c7cdfd782d826a/xmake/core/package/package.lua#L992